web/styles/popovers.css

```diff
@@ -674,7 +679,8 @@ ul.popover-group-menu-member-list {
                         padding-right: 10px;
                     }
 
-                    &.remove_subscription {
+                    &.remove_subscription,
+                    &.remove_member {
```

Comment:
Probably as a follow-up we should rename these along the lines of https://github.com/zulip/zulip/pull/32527/commits/3f8c27d4b182eb6a78eaba4a8bb9ad62ef3f82e1#r1868548937 to be more globally unique.